# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1330209044 18000
#      Sat Feb 25 17:30:44 2012 -0500
# Node ID c4a2eac243b4fd3e45a048d276887150052e0dc8
# Parent  b9630dfc4fd93437118f39397171c0065ba9374f
mesh.m: Modify demos to allow conversion to Matlab.

diff --git a/scripts/plot/mesh.m b/scripts/plot/mesh.m
--- a/scripts/plot/mesh.m
+++ b/scripts/plot/mesh.m
@@ -66,25 +66,22 @@ endfunction
 %!demo
 %! clf ();
 %! x = logspace(0,1,11);
 %! z = x'*x;
 %! mesh (x, x, z, z.^2);
 %! xlabel xlabel
 %! ylabel ylabel
 %! zlabel "linear scale"
-%! if (strcmp (get (gcf, "__graphics_toolkit__"), "gnuplot"))
-%!   title ("Gnuplot's mesh color is correct")
-%! endif
 
 %!demo
 %! clf ();
 %! x = logspace(0,1,11);
 %! z = x'*x;
 %! mesh (x, x, z, z.^2);
 %! set (gca, "zscale", "log")
 %! xlabel xlabel
 %! ylabel ylabel
 %! zlabel "log scale"
 %! if (strcmp (get (gcf, "__graphics_toolkit__"), "gnuplot"))
-%!   title ({"Gnuplot's mesh color is wrong", "This a Gnuplot bug"})
+%!   title ({"Gnuplot: mesh color is wrong", "This a Gnuplot bug"})
 %! endif
 
